README.md

```diff
@@ -430,6 +432,18 @@ On the coordinator node:
 fab run.tpch_automate:tpch_q1.ini,connectionURI='postgres://citus:dwVg70yBfkZ6hO1WXFyq1Q@c.fhhwxh5watzbizj3folblgbnpbu.db.citusdata.com:5432/citus?sslmode\=require'
 ```
 
+## <a name="valgrind"></a> Running Valgrind Tests
+On the coordinator node:
+
+```bash
+# Note that we do not call use.citus(or enterprise) yet as we will do it in run.valgrind task
```

Comment:
Why dont we use use.citus or enterprise here? We can use the existing logic instead of creating a new one

Reply:
For some tests, we call directly run command as I see in README. Other than this, I wanted `run.valgrind` task to be aware of the path we are working on in order to move the logs from there. Hence, I parameterized `run.valgrind` so it also specifies repository. As I already parameterized it, using `use.citus/enterprise` may result in conflicts and seems redundant to me. This why I utilize `use.citus/enterprise` in `run.valgrind`.

Reply:
That makes sense, applied necessary changes